### ISSUE TITLE
[feature] Setting for standard view

### ISSUE THREAD

**Issue by lenowac** [etesync/etesync-web#210]
I like my calender in weekly view, just as some people probably prefer agenda or daily as their standard. 

I think a setting for that would be neat :)

**Comment by tasn** [comment]
Yeah. It should probably also save the last one if it doesn't already.
